client/js/renderPreview.js

```diff
@@ -61,3 +61,44 @@ function handleImageInPreview(content, container) {
 		});
 	}
 }
+
+/* Image viewer */
+
+// FIXME Remove #input focus when this is open
```

Comment:
If you aren't going to fix this in this PR (which I'm ok with) then stick up an issue on github for it and put in the number here. Otherwise this will get missed. I hate random "FIXME" and "TODO" s that don't have any number attached to them